### PR TITLE
implemented basic file watcher

### LINKED ISSUE #2: Watch file changes
Use watcher and update tests in UI in realtime

### PR DESCRIPTION
**Pull request by DavertMik**
Watch for file updates and send notifications on changed tests via websockets
Resolves #2

### CODE REVIEW

File: lib/api/list-scenarios.js [hunk 1]
```diff
@@ -35,4 +43,43 @@ module.exports = (req, res) => {
     name: config.get('name'),
     features
   });
+
+  function watchFileChanges() {
+    const socket = require('socket.io-client')(WS_URL);
+
+    // does not handle deleted files... yet...
+    chokidar.watch(path.join(codecept_dir, config.get('tests')), {ignored: /(^|[\/\\])\../}).on('all', (event, path) => {
+      console.log('Changed ', path);
```
Comment (hubidu) [inline lib/api/list-scenarios.js]: I like using debug() because I can turn it on when needed

Comment (DavertMik) [reply lib/api/list-scenarios.js]: Well, this can be important user information, so I think it should be printed.

**Comment by DavertMik** [comment]
> Looking forward to see how it works out.

Are you merging it, or you will go with your own branch?

**Comment by hubidu** [comment]
I merged it locally, but I still need to fix some conflicts

**Pull request closed without merging**